script.js

```diff
@@ -9,4 +10,163 @@ menu.onclick = () => {
+    type: 1,
+  },
+
+  {
+    id: 4,
+    title: 'Multi-Post Stories',
+    farme: ['Uber', 'Lead Developer', 2018],
+    image: './Images/4.png',
+    description_: 'A daily selection of privately personalized reads; no accounts or sign-ups required',
+    description: `Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+                  when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+                  It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent`,
+    type: 2,
+  },
+];
+function createWorkCard() {
+  works.forEach((work) => {
+    const sampleTemplateModal = `
+  <section class="work-container${work.type}">
+  <img src=${work.image} alt="">
+  <div class="work">
+    <h2>${work.title}</h2>
+    <div class="facebook"> <p>${work.farme[0]}</p>
+      <ul>
+         <li>${work.farme[1]}</li>
+         <li>${work.farme[2]}</li>
+      </ul>
+    </div>
+
+  <p class="work-p">${work.description}</p>
+  <ul class="work-list">
+     <li><a href=""> html</a></li>
+     <li class="desktop_link"><a href=""> Ruby on rails</a></li>
+     <li><a href=""> css</a></li>
+     <li><a href=""> javaScript</a></li>
+   </ul>
+   <button class="btn modal-btn" id="${work.id}">See project</button>
+ </div>
+</section>
+`;
+    const element = document.querySelector('.my-works');
+    element.innerHTML += sampleTemplateModal;
+  });
+}
+createWorkCard();
+/* -----------------------popup window-------------------------*/
+
+const openPopButtons = document.querySelectorAll('.modal-btn');
+const overlay = document.querySelector('#overlay');
+function createModal(id, title, image, description, farme) {
+  const sampleTemplateModal = `
+  <section class="modal-work">
+        <div class="header-wrapper">
+            <h2>${title}</h2>
+             <a href="#" id="crossBtnImg">
+                <img src="./Images/Cancel.png" alt="" class="data-close-button">
+            </a>
+        </div>
+        <div class="facebook"> <p>${farme[0]}</p>
+           <ul>
+              <li>${farme[1]}</li>
+              <li>${farme[2]}</li>
+           </ul>
+        </div>
+        <img src="${image}" alt="" class="modal-image">
+        <div class="info-skill">
+          <p class="modal-text">${description}</p>
+
+            <div class="skill-button">
+                <ul class="work-list">
+                  <li><a href=""> html</a></li>
+                  <li><a href=""> css</a></li>
+                  <li><a href=""> javaScript</a></li>
+               <li class="desktop_link"><a href=""> Ruby</a></li>
+               <li class="desktop_link"><a href=""> Bootstrap</a></li>
+                  </ul> <hr>
+              <div class="button-wrapper">
+                  <button class="btn pop-up-btn "> <span>See Live</span><a href="#"><img src="./Images/Icon.png" alt=""></a>
```

Comment:
- [ ] The image of your project card is not currently resizing appropriately it looks really deformed in the desktop version. To fix it I would suggest you to use `aspect-ration: 2/1` instead of property `height`.